packages/assets/css/src/scss/_components/_modal/_elements/_modal-body.scss

```diff
@@ -14,7 +14,6 @@
 		@include block($block) {
 			@include element(body) {
 				padding: $spacing-xxl;
-				overflow-x: auto;
```

Comment:
why did we remove it? I think without this when content will be too much it won't scroll.

Reply:
@creador-dev I removed it to solve this issue: 

https://github.com/wpmudev/sui-react/assets/39377174/0050f39d-4728-4b46-aaed-767f70cb1dc3

for the x axis I don't think we will need to set it to auto as the items should always stack ( as per design ) 

but for the y axis we need it when the modal hight become bigger than the screen, but still overflow-y will have the issue above, absolute elements should not make the scroll bar appear

Reply:
let me know if you have an idea to solve this

Reply:
@abder removing it causes this error when the content is too long. @emgk what should we do here? 
The solution I can think of is to place the select dropdown HTML last item in the body. The same as what we faced with the tooltip and solution we discussed we have to implement here as well I think.
![Screenshot 2024-01-16 at 2 02 31 PM](https://github.com/wpmudev/sui-react/assets/40248406/74880e33-1090-491b-b30e-0cc04ed8477b)

Reply:
@creador-dev Yes that was my point regarding y access, but isn't it that the select currenly is placed at the very bottom? and what that has to change?

Reply:
@abder No I mean the menu part should be outside of the modal at the very bottom like other libraries do.

Reply:
@creador-dev you mean injecting it to the root document using react portal? if so, it will be tricky to detect type of the component and move out of the modal as modal body content is dynamic. Also whole modal should be scrollable not just modal body, select dropdown area should be absolute and not shifting the content downwards.

Can we make modal container part scrollable and assign `(scrollable height - margin on top and bottom)` to the modal (not using JS, i think its possible with pure CSS only). please check popup in dev hub, i've made these improvements there in past, so would be helpful. :)

![image](https://github.com/wpmudev/sui-react/assets/14994452/2f78a272-b68c-4db4-85f6-064a123763eb)

Reply:
will be fixed in another branch.